gallery/v2_transforms/plot_transforms_v2.py

```diff
@@ -99,10 +99,9 @@
     format="XYXY", canvas_size=img.shape[-2:])
 
 transforms = v2.Compose([
-    v2.RandomPhotometricDistort(),
-    v2.RandomIoUCrop(),
-    v2.RandomHorizontalFlip(p=0.5),
-    v2.SanitizeBoundingBoxes(),
+    v2.RandomResizedCrop(size=(224, 224), antialias=True),
+    v2.RandomPhotometricDistort(p=1),
+    v2.RandomHorizontalFlip(p=1),
```

Comment:
I was actually mis-using `SanitizeBoundingBoxes()` as I wasn't passing a label, so I should have set `labels_getter` to None (see non-regression test below). I decided to just remove SanitizeBoundingBoxes here because that would require making the input a dict, also forcing to change the narrative below. It's OK, SanitizeBoundingBoxes will be clearly used and mentioned in the e2e example that I'm working on now.